addons/l10n_in_withholding/models/l10n_in_section_alert.py

```diff
@@ -32,7 +32,7 @@ class L10nInSectionAlert(models.Model):
     @api.depends('tax_source_type')
     def _compute_display_name(self):
         for record in self:
-            record.display_name = f"{record.tax_source_type.upper()} {record.name}"
+            record.display_name = f"{record.tax_source_type and record.tax_source_type.upper()} {record.name}"
```

Comment:
if there is no `tax_source_type` the display name will begin with a space. Is that what we want?

Reply:
@vivp-odoo That is not nice "False name".    
```suggestion
            record.display_name = f"{record.tax_source_type.upper()} {record.name}" if record.tax_source_type else f"{record.name}"
```